bool walk_enabled
int64 step_length
int64 side_length
float32 angle
---
bool walk_enabled
